internal/commands/pause_notify.go

```diff
@@ -73,15 +75,61 @@ func PauseNotifyIncidentByDialog(
 	repository model.Repository,
 	incidentDetails bot.DialogSubmission,
 ) error {
+
+	var (
+		channelID             = incidentDetails.Channel.ID
+		userID                = incidentDetails.User.ID
+		submissions           = incidentDetails.Submission
+		pauseNotifyTimeText   = submissions.PauseNotifyTime
+		pauseNotifyReasonText = submissions.PauseNotifyReason
+
+		pauseNotifyTime time.Time
+	)
+
+	days, err := strconv.Atoi(pauseNotifyTimeText)
+	if err != nil {
+		logger.Error(
+			ctx,
+			"command/pauseNotify.PauseNotifyIncidentByDialog strconv.Atoi ERROR",
+			log.NewValue("channelID", channelID),
+			log.NewValue("pauseNotifyTimeText", pauseNotifyTimeText),
+			log.NewValue("error", err),
+		)
+
+		PostErrorAttachment(ctx, client, logger, channelID, userID, err.Error())
+		return err
+	}
+
+	pauseNotifyTime = time.Now().AddDate(0, 0, days)
+
 	logger.Info(
 		ctx,
 		"command/pauseNotify.PauseNotifyIncidentByDialog",
+		log.NewValue("pauseNotifyTimeText", pauseNotifyTimeText),
+		log.NewValue("pauseNotifyReasonText", pauseNotifyReasonText),
+		log.NewValue("pauseNotifyTime", pauseNotifyTime),
 		log.NewValue("incidentDetails", incidentDetails),
```

Comment:
If you are logging all the incidentDetails here, there is no reason to log the above three.

Reply:
Removed :)